src/documents/Enactors.md

```diff
@@ -23,18 +23,18 @@ Each of these is explained below.
 
 ### Grade ###
 
-Enactors must be defined using the `fluid.prefs.enactors` [grade](ComponentGrades.md), as shown in the following code block:
+Enactors must be defined using the `fluid.prefs.enactor` [grade](ComponentGrades.md), as shown in the following code block:
 ```javascript
 fluid.defaults("my.pref.enactor", {
-    gradeNames: ["fluid.prefs.enactors", "autoInit"],
+    gradeNames: ["fluid.prefs.enactor", "autoInit"],
     ...
 });
 ```
 
-Enactors are, by default, [model components](tutorial-gettingStartedWithInfusion/ModelComponents.md) and [evented components](tutorial-gettingStartedWithInfusion/EventedComponents.md), so they automatically provides support for a model and for events. If other support is needed, other grades can be added. For example, if the enactor will be operating on the DOM, the [`fluid.viewComponent`](https://github.com/fluid-project/infusion/blob/infusion-1.5/src/framework/core/js/FluidView.js#L34-L36) grade should be used, and the `selectors` option should be provided, as shown in the following example:
+Enactors are, by default, [model components](tutorial-gettingStartedWithInfusion/ModelComponents.md) and [evented components](tutorial-gettingStartedWithInfusion/EventedComponents.md), so they automatically provides support for a model and for events. If other support is needed, other grades can be added. For example, if the enactor will be operating on the DOM, the [`fluid.viewRelayComponent`](https://github.com/fluid-project/infusion/blob/infusion-1.5/src/framework/core/js/FluidView.js#L40-L42) grade should be used, and the `selectors` option should be provided, as shown in the following example:
```

Comment:
What should we do about this link that's hard-coded to 1.5? Should we change it to master for now?